index.js

```diff
@@ -72,7 +72,7 @@ module.exports = options => {
 		}, 200);
 		window.addEventListener('unhandledrejection', event => {
 			event.preventDefault();
-			rejectionHandler(event.reason);
+			rejectionHandler(event.reason || event.detail.reason);
```

Comment:
Instead of covering all the possible unique cases of different event.reasons should this not become an option? IE do something like this?

options.eventReasonPath = ['reason']; // default

`rejectionHandler(get(event, options.eventReasonPath))` ??

Reply:
No, libraries should just follow the standard.